### PR TITLE
docs: translate introduction of escape hatches

### PR DESCRIPTION
**Pull request by eriansha**
Closes https://github.com/reactjs/id.react.dev/issues/386

## Description

Translate the escape hatches page.
Page URL: https://react.dev/learn/escape-hatches

### CODE REVIEW

File: src/content/learn/escape-hatches.md [hunk 1]
```diff
@@ -93,15 +93,15 @@ export default function Form() {
...
 
-Some components need to synchronize with external systems. For example, you might want to control a non-React component based on the React state, set up a server connection, or send an analytics log when a component appears on the screen. Unlike event handlers, which let you handle particular events, *Effects* let you run some code after rendering. Use them to synchronize your component with a system outside of React.
+Beberapa komponen perlu menyinkronkan dengan sistem eksternal. Misalkan, Anda mungkin ingin mengontrol komponen *non-React* berdasarkan *state* React, mengatur koneksi server, atau mengirim log analitik ketika sebuah komponen muncul di layar. Tidak seperti *event handlers*, yang memungkinkan Anda menangani *events* tertentu, *Effects* memungkinkan Anda menjalankan beberapa kode setelah *render*. Gunakan *Effects* ini untuk menyinkronkan komponen Anda dengan sistem di luar React.
 
-Press Play/Pause a few times and see how the video player stays synchronized to the `isPlaying` prop value:
+Menekan tombol *Play*/*Pause* beberapa kali dan lihat bagaimana pemutar video tetep disinkronkan dengan nilai prop `isPlaying`:
```
Comment (resir014) [inline src/content/learn/escape-hatches.md:104]:
```suggestion
Tekan tombol *Play*/*Pause* beberapa kali dan lihat bagaimana pemutar video tetep disinkronkan dengan nilai prop `isPlaying`:
```

"press" di sini adalah kata perintah, bukan kata kerja.

**Review by resir014 — COMMENTED** [review]
One minor non-blocking change required. Please check them out. @eriansha

File: src/content/learn/escape-hatches.md (hunk 1 above)
Comment (eriansha) [reply src/content/learn/escape-hatches.md]: addresses in a33ff80. thanks for the feedback!

File: src/content/learn/escape-hatches.md [hunk 2]
```diff
@@ -4,32 +4,32 @@ title: Escape Hatches
...
 
-* [How to "remember" information without re-rendering](/learn/referencing-values-with-refs)
-* [How to access DOM elements managed by React](/learn/manipulating-the-dom-with-refs)
-* [How to synchronize components with external systems](/learn/synchronizing-with-effects)
-* [How to remove unnecessary Effects from your components](/learn/you-might-not-need-an-effect)
-* [How an Effect's lifecycle is different from a component's](/learn/lifecycle-of-reactive-effects)
-* [How to prevent some values from re-triggering Effects](/learn/separating-events-from-effects)
-* [How to make your Effect re-run less often](/learn/removing-effect-dependencies)
-* [How to share logic between components](/learn/reusing-logic-with-custom-hooks)
+* [Bagaimana cara "mengingat" informasi tanpa harus melakukan rendering kembali](/learn/referencing-values-with-refs)
```
Comment (mhaidarhanif) [inline src/content/learn/escape-hatches.md:13]: Almost forgot, should rendering be italicized even inside a link?

```suggestion
* [Bagaimana cara "mengingat" informasi tanpa harus melakukan *rendering* kembali](/learn/referencing-values-with-refs)
```

File: src/content/learn/escape-hatches.md [hunk 3]
```diff
@@ -4,32 +4,32 @@ title: Escape Hatches
...
 
-When you want a component to "remember" some information, but you don't want that information to [trigger new renders](/learn/render-and-commit), you can use a *ref*:
+Ketika Anda ingin sebuah komponen "mengingat" beberapa informasi, tapi Anda tidak ingin informasi tersebut [memicu *render* baru](/learn/render-and-commit), Anda dapat menggunakan *ref*:
 
 ```js
 const ref = useRef(0);
 ```
 
-Like state, refs are retained by React between re-renders. However, setting state re-renders a component. Changing a ref does not! You can access the current value of that ref through the `ref.current` property.
+Sama seperti *state*, *refs* disimpan oleh React diantara pe-*render*-an ulang. Namun, mengatur *state* menyebabkan komponen di-*render* ulang. Mengganti sebuah *ref* tidak! Anda dapat mengakses nilai saat ini dari *ref* tersebut melalui properti `ref.current`.
```
Comment (mhaidarhanif) [inline src/content/learn/escape-hatches.md:32]: Need to adapt the translation to make it easier to understand.


```suggestion
Sama seperti *state*, *refs* disimpan oleh React diantara pe-*render*-an ulang. Namun, mengatur *state* menyebabkan komponen di-*render* ulang. Mengganti sebuah *ref* tidak menyebabkan itu! Anda dapat mengakses nilai saat ini dari *ref* tersebut melalui properti `ref.current`.
```

File: src/content/learn/escape-hatches.md [hunk 4]
```diff
@@ -145,7 +145,7 @@ video { width: 250px; }
 
 </Sandpack>
 
-Many Effects also "clean up" after themselves. For example, an Effect that sets up a connection to a chat server should return a *cleanup function* that tells React how to disconnect your component from that server:
+Banyak *Effects* juga melakukan "pemberishan" setelah mereka selesai. Misalkan, sebuah *Effect* yang mengatur koneksi ke *chat server* harus mengembalikan fungsi pembersih (*cleanup function*) yang memberi tahu React bagaimana cara memutuskan koneksi komponen Anda dari *server* tersebut:
```
Comment (mhaidarhanif) [inline src/content/learn/escape-hatches.md:148]: Fix typo


```suggestion
Banyak *Effects* juga melakukan "pembersihan" setelah mereka selesai. Misalkan, sebuah *Effect* yang mengatur koneksi ke *chat server* harus mengembalikan fungsi pembersih (*cleanup function*) yang memberi tahu React bagaimana cara memutuskan koneksi komponen Anda dari *server* tersebut:
```

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
Some improvements

**Comment by resir014** [comment]
Hi @eriansha, do you have time to make the requested changes?

**Comment by eriansha** [comment]
> Hi @eriansha, do you have time to make the requested changes?

sure, sorry for delayed response.
i might create the changes on tommorrow 🙇🏻‍♂️

**Comment by eriansha** [comment]
resolving @mhaidarhanif's comment. thank you to check small detail that i missed out 😄

**Review by mhaidarhanif — APPROVED** [review]
@eriansha You're welcome

**Pull request merged**